osu.Game/Modes/UI/StandardHealthDisplay.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+
+using OpenTK;
+using OpenTK.Graphics;
+using osu.Framework.Allocation;
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.Transforms;
+using osu.Game.Graphics;
+
+namespace osu.Game.Modes.UI
+{
+    internal class StandardHealthDisplay : HealthDisplay

```

Comment:
not sure about making all these classes internal. might as well leave them public (they can be used in visualtests then, if we decide to).